src-java/testing/performance-tests/src/test/groovy/org/openkilda/performancetests/BaseSpecification.groovy

```diff
@@ -125,4 +127,19 @@ class BaseSpecification extends Specification {
         //setup with empty body in order to trigger a SETUP invocation, which is intercepted in several extensions
         //this can have implementation if required
     }
+
+    Switch pickRandom(List<Switch> switches) {
+        switches[new Random().nextInt(switches.size())]
+    }
+
+    void deleteFlows( List<FlowExtended> flows) {
+        flows.each { it.sendDeleteRequest() }
+        def waitTime = northbound.getAllFlows().size()
```

Comment:
waitTime is equal to flow size.
just in case, is it correct?

Reply:
yes, this logic was used before, but I modified it to wait for 'existing flows' seconds. This is mostly for the case when we have 1000 or 300+ flows to decrease the time of waiting in case of any errors (anyway we have topology purge at the beginning of each spec).